src/library/dao/trackdao.cpp

```diff
@@ -1905,6 +1940,7 @@ bool TrackDAO::detectMovedTracks(
             const auto nextSuffixMatch =
                     matchStringSuffix(nextTrackLocation, oldTrackLocation);
             DEBUG_ASSERT(nextSuffixMatch >= filename.length());
+            // document this
```

Comment:
Any chance you can address this? Or maybe remove this comment otherwise

Reply:
I added the documentation.